tests/regression/chatbot/compare_bm25_fallback_paths.py

```
@@ -0,0 +1,430 @@
+        "--cases",
+        str(args.cases),
+        "--top-k",
+        str(args.top_k),
+        "--answer-top-k",
+        str(args.answer_top_k),
+    ]
+    if args.max_cases:
+        cmd.extend(["--max-cases", str(args.max_cases)])
+
+    started = time.perf_counter()
+    completed = subprocess.run(cmd, env=env, capture_output=True, text=True, check=False)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)